Tests/Marketplace/search_and_install_packs.py

```diff
@@ -137,35 +126,41 @@ def search_pack(client, prints_manager, pack_display_name, thread_index, lock):
     try:
         # make the search request
         response_data, status_code, _ = demisto_client.generic_request_func(client,
-                                                                            path='/contentpacks/marketplace/search',
-                                                                            method='POST',
-                                                                            body={"packsQuery": pack_display_name},
+                                                                            path=f'/contentpacks/marketplace/{pack_id}',
+                                                                            method='GET',
                                                                             accept='application/json',
                                                                             _request_timeout=None)
 
         if 200 <= status_code < 300:
             result_object = ast.literal_eval(response_data)
-            search_results = result_object.get('packs', [])
-            pack_data = get_pack_data_from_results(search_results, pack_display_name)
-            if pack_data:
-                print_msg = 'Found pack {} in bucket!\n'.format(pack_display_name)
+
+            if result_object and result_object.get('currentVersion'):
+                print_msg = 'Found pack "{}" by its ID "{}" in bucket!\n'.format(pack_display_name, pack_id)
                 prints_manager.add_print_job(print_msg, print_color, thread_index, LOG_COLORS.GREEN)
                 prints_manager.execute_thread_prints(thread_index)
+
+                pack_data = {
+                    'id': result_object.get('id'),
+                    'version': result_object.get('currentVersion')
+                }
                 return pack_data
 
             else:
-                print_msg = 'Did not find pack {} in bucket.\n'.format(pack_display_name)
+                print_msg = 'Did not find pack "{}" by its ID "{}" in bucket.\n'.format(pack_display_name, pack_id)
```

Comment:
use f-strings son

Reply:
We will have some like this and some like this... I don't think it's better... it is more confusing.

Reply:
you already have it mixed, you modified the code yourself to be like that

Reply:
Hhhhh no I did not but yalla

Reply:
Yalla changed